znai-core/src/main/java/com/twosigma/znai/search/GlobalSearchEntries.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 import java.util.Set;
 
-@JacksonXmlRootElement(localName = "mdoc")
+@JacksonXmlRootElement(localName = "znai")
```

Comment:
Some of old directly uploaded documents still have <mdoc>.
The change might break the documents.

Reply:
I may have misunderstood @jyeakel in #262. On Coveo side you could add two rules/crawlers to handle both cases. 
@jyeakel what do you think? Without adding them the worst case I can see is indexes won't be restored if Coveo need to re-index everything from scratch. Unless Coveo decides that content is not present anymore and will delete indexing... hmm.

Reply:
I checked Coveo config for mdoc and we have both mdoc/entry and znai/entry defined.
We are good to go.